charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -18,7 +18,7 @@ java:
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
     DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-399.service.core-compute-preview.internal/
+    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-414.service.core-compute-preview.internal/
```

Comment:
Once the user profile PR is released all ok, pls don't forget to revert this line and wire this PR with AAT.

Reply:
Yes, it will be reverted back after the successful completion of QA.